scripts/infra/cloud-instance.sh

```diff
@@ -13,6 +13,17 @@ else
     exit 1
 fi
 
+ensure_system_pkgs() {
+    # shellcheck source=/etc/os-release
+    source /etc/os-release
+    if [ $ID == "fedora" ] || [ $ID == "rhel" ]; then
+        PKG_LIST=(git gcc python-devel python-virtualenv krb5-devel openldap-devel aws)
+        sudo dnf install "${PKG_LIST[@]}"
```

Comment:
Should this be with -y?  Or do you want confirmation?

Reply:
should have `-y` for sure